cmd/workload-controller-manager/config/controllerconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-    "reportHealthIntervalInSecond": 10,
+    "reportHealthIntervalInSecond": 60,
```

Comment:
This is not an ultimate fix.
1. TTL is 100 seconds. One miss report could just remove the instance from the registry and cause hash key range updates.
2. 100 seconds to find controller instance is missing is not bearable.

Reply:
Updated TTL to 300

Reply:
Can this be changed back to 10s and TTL keep as 30s?

Reply:
I am asking because it looks like in production, WorkloadControllerManager will be auto restarted if it exits. In that case, it will take long time for unlock.